app/user_auth.py

```diff
@@ -14,10 +14,37 @@
 from lib.database.reservations import get_reservation_for_user
 from lib.database.promotion_keys import pop_promotion_key
 from lib.database.showtimes import get_showtime
+from .lib.email_sender import send_reminder
+from .lib.database.showtimes import get_showtimes
+from .lib.database.reservations import get_reservations
+from lib.basehandler import secured
 from lib.email_sender import send_confirmation
 from lib.basehandler import BaseHandler
 
 
+@secured
+class UserReminder(BaseHandler):
+    _ioloop = ioloop.IOLoop().instance()
+
+    @web.asynchronous
+    @gen.coroutine
+    def get(self):
+        all_reservations = yield get_reservations()
+        all_showtimes = yield get_showtimes()
+        for reservation in all_reservations:
+            show_id = reservation['showtime_id']
+            date_str = filter(
```

Comment:
remove this and have `show_meta = yield get_showtime(show_id)` and the string date is in there. No need to resolve problems we've already solved.